tests/system/HTTP/RedirectResponseTest.php

```diff
@@ -24,8 +25,10 @@ protected function setUp()
 
 		$_SERVER['REQUEST_METHOD'] = 'GET';
 
-		$this->config          = new App();
-		$this->config->baseURL = 'http://example.com';
+		$this->config            = new App();
+		$this->config->baseURL   = 'http://example.com';
+		$this->config->indexPage = '';
```

Comment:
Setting indexPage to empty breaks the form & HTML helper tests big-time!

Reply:
Why does changing the local `App` instance here influence other tests?

Reply:
And how to resolve this? I need a different `indexPage` for the test

Reply:
Don't change it in `setUp()`, set the specific values you need only on the test where they differ.

Reply:
Ohhh I get it now, you were trying to "undo" the injection of your test. Let me think of another way to approach this...